doc/filters/data_uri.rst

```diff
@@ -32,8 +32,8 @@ The ``data_uri`` filter generates a URL using the data scheme as defined in RFC
 
         $ composer require twig/html-extra
 
-    Then, use the ``twig/extra-bundle`` on Symfony projects or add the extension
-    explicitly on the Twig environment::
+    On Symfony projects, install the `twig/extra-bundle`_.
```

Comment:
I would remove the link to Github's README and give the command here (as done above for the html-extra package)

Reply:
Like this?
This line should probably go onto *all* extra-bundle filters. Do you want to create an include or something? If no, I'd commit it manually - is there a list of all those "extra" filters somewhere?